nibabel/tests/test_volumeutils.py

```diff
@@ -1245,3 +1248,42 @@ def read(self, n_bytes):
                  'Expected {0} bytes, got {1} bytes from {2}\n'
                  ' - could the file be damaged?'.format(
                      11390625000000000000, 0, 'object'))
+
+
+def test__ftype4scaled_finite_warningfilters():
```

Comment:
Could you explain a little more why you need the big array here?

Reply:
This was to force the job to take long enough I could reliably produce a race that resulted in an error condition.